ansible.cfg

```diff
@@ -1,2 +0,0 @@
-[defaults]
-roles_path = ~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles:roles
```

Comment:
roles_path defaults to:
```
default: ~/.ansible/roles:/usr/share/ansible/roles:/etc/ansible/roles
```

So we are completely removing support from the installer git tree for running the roles directly from this?

We may have agreed this at the meeting, but I forgot.

As this stands right now, it will slow down my development.

And it is breaking change, with no issue or changedlog, for installing Pulp the old way.

Reply:
I need to think about this. Sadly, last time i checked, there is no equivalent to `pip install -e` for collections.

Reply:
instead of roles_path, i added collections_path, so you should be able to run things locally again (after `make install`).

Reply:
Thank you!